docs/niscope/functions.rst

```diff
@@ -350,13 +350,13 @@ niscope.Session methods
         Device <REPLACE_DRIVER_SPECIFIC_URL_1(features_supported_main)>`__ for
         more information.
 
-        Default value: VI\_TRUE
+        Default value: VI_TRUE
```

Comment:
Food for thought: replace VI_TRUE with true

Reply:
Done, let me know if I should keep it.